Startup fails: `Cannot find module 'electron'`

## Describe the bug

The IDE fails to start with the error:

```text
$ ./Arduino\ IDE.exe

Starting backend process. PID: 3844
Failed to start the backend application:
Error: Cannot find module 'electron'
Require stack:
- C:\ide 2\arduino-ide_2.0.0-rc3-snapshot-5ab3a74_Windows_64bit\resources\app\node_modules\arduino-ide-extension\lib\node\grpc-client-provider.js
- C:\ide 2\arduino-ide_2.0.0-rc3-snapshot-5ab3a74_Windows_64bit\resources\app\node_modules\arduino-ide-extension\lib\node\core-client-provider.js
- C:\ide 2\arduino-ide_2.0.0-rc3-snapshot-5ab3a74_Windows_64bit\resources\app\node_modules\arduino-ide-extension\lib\node\library-service-server-impl.js
- C:\ide 2\arduino-ide_2.0.0-rc3-snapshot-5ab3a74_Windows_64bit\resources\app\node_modules\arduino-ide-extension\lib\node\arduino-ide-backend-module.js
- C:\ide 2\arduino-ide_2.0.0-rc3-snapshot-5ab3a74_Windows_64bit\resources\app\src-gen\backend\server.js
- C:\ide 2\arduino-ide_2.0.0-rc3-snapshot-5ab3a74_Windows_64bit\resources\app\src-gen\backend\main.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:934:15)
    at Module._load (node:internal/modules/cjs/loader:779:27)
    at Function.c._load (node:electron/js2c/asar_bundle:5:13331)
    at Module.require (node:internal/modules/cjs/loader:1006:19)
    at require (node:internal/modules/cjs/helpers:93:18)
    at Object.<anonymous> (C:\ide 2\arduino-ide_2.0.0-rc3-snapshot-5ab3a74_Windows_64bit\resources\app\node_modules\arduino-ide-extension\lib\node\grpc-client-provider.js:14:20)
    at Module._compile (node:internal/modules/cjs/loader:1110:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1140:10)
    at Module.load (node:internal/modules/cjs/loader:982:32)
    at Module._load (node:internal/modules/cjs/loader:823:12) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    'C:\\ide 2\\arduino-ide_2.0.0-rc3-snapshot-5ab3a74_Windows_64bit\\resources\\app\\node_modules\\arduino-ide-extension\\lib\\node\\grpc-client-provider.js',
    'C:\\ide 2\\arduino-ide_2.0.0-rc3-snapshot-5ab3a74_Windows_64bit\\resources\\app\\node_modules\\arduino-ide-extension\\lib\\node\\core-client-provider.js',
    'C:\\ide 2\\arduino-ide_2.0.0-rc3-snapshot-5ab3a74_Windows_64bit\\resources\\app\\node_modules\\arduino-ide-extension\\lib\\node\\library-service-server-impl.js',
    'C:\\ide 2\\arduino-ide_2.0.0-rc3-snapshot-5ab3a74_Windows_64bit\\resources\\app\\node_modules\\arduino-ide-extension\\lib\\node\\arduino-ide-backend-module.js',
    'C:\\ide 2\\arduino-ide_2.0.0-rc3-snapshot-5ab3a74_Windows_64bit\\resources\\app\\src-gen\\backend\\server.js',
    'C:\\ide 2\\arduino-ide_2.0.0-rc3-snapshot-5ab3a74_Windows_64bit\\resources\\app\\src-gen\\backend\\main.js'
  ]
}
C:\ide 2\arduino-ide_2.0.0-rc3-snapshot-5ab3a74_Windows_64bit\resources\app\node_modules\@theia\core\lib\node\main.js:22
    throw reason;
    ^

Error: Cannot find module 'electron'
Require stack:
- C:\ide 2\arduino-ide_2.0.0-rc3-snapshot-5ab3a74_Windows_64bit\resources\app\node_modules\arduino-ide-extension\lib\node\grpc-client-provider.js
- C:\ide 2\arduino-ide_2.0.0-rc3-snapshot-5ab3a74_Windows_64bit\resources\app\node_modules\arduino-ide-extension\lib\node\core-client-provider.js
- C:\ide 2\arduino-ide_2.0.0-rc3-snapshot-5ab3a74_Windows_64bit\resources\app\node_modules\arduino-ide-extension\lib\node\library-service-server-impl.js
- C:\ide 2\arduino-ide_2.0.0-rc3-snapshot-5ab3a74_Windows_64bit\resources\app\node_modules\arduino-ide-extension\lib\node\arduino-ide-backend-module.js
- C:\ide 2\arduino-ide_2.0.0-rc3-snapshot-5ab3a74_Windows_64bit\resources\app\src-gen\backend\server.js
- C:\ide 2\arduino-ide_2.0.0-rc3-snapshot-5ab3a74_Windows_64bit\resources\app\src-gen\backend\main.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:934:15)
    at Module._load (node:internal/modules/cjs/loader:779:27)
    at Function.c._load (node:electron/js2c/asar_bundle:5:13331)
    at Module.require (node:internal/modules/cjs/loader:1006:19)
    at require (node:internal/modules/cjs/helpers:93:18)
    at Object.<anonymous> (C:\ide 2\arduino-ide_2.0.0-rc3-snapshot-5ab3a74_Windows_64bit\resources\app\node_modules\arduino-ide-extension\lib\node\grpc-client-provider.js:14:20)
    at Module._compile (node:internal/modules/cjs/loader:1110:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1140:10)
    at Module.load (node:internal/modules/cjs/loader:982:32)
    at Module._load (node:internal/modules/cjs/loader:823:12) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    'C:\\ide 2\\arduino-ide_2.0.0-rc3-snapshot-5ab3a74_Windows_64bit\\resources\\app\\node_modules\\arduino-ide-extension\\lib\\node\\grpc-client-provider.js',
    'C:\\ide 2\\arduino-ide_2.0.0-rc3-snapshot-5ab3a74_Windows_64bit\\resources\\app\\node_modules\\arduino-ide-extension\\lib\\node\\core-client-provider.js',
    'C:\\ide 2\\arduino-ide_2.0.0-rc3-snapshot-5ab3a74_Windows_64bit\\resources\\app\\node_modules\\arduino-ide-extension\\lib\\node\\library-service-server-impl.js',
    'C:\\ide 2\\arduino-ide_2.0.0-rc3-snapshot-5ab3a74_Windows_64bit\\resources\\app\\node_modules\\arduino-ide-extension\\lib\\node\\arduino-ide-backend-module.js',
    'C:\\ide 2\\arduino-ide_2.0.0-rc3-snapshot-5ab3a74_Windows_64bit\\resources\\app\\src-gen\\backend\\server.js',
    'C:\\ide 2\\arduino-ide_2.0.0-rc3-snapshot-5ab3a74_Windows_64bit\\resources\\app\\src-gen\\backend\\main.js'
  ]
}
```

## To Reproduce

1. Attempt to start the Arduino IDE.

🐛 It does not start.

If you run it from the command line, you will see the output I provided above.

## Expected behavior

IDE starts up

## Desktop

- OS: Windows 10, Ubuntu 20.04
- Version: 2.0.0-rc3-snapshot-5ab3a74
CLI Version: 0.21.0 [10107d24]

## Additional context

I bisected the bug to https://github.com/arduino/arduino-ide/commit/5ab3a747a6e8bf551b5c0e59a98154d387e0a200 (it does not occur in the build from the previous commit https://github.com/arduino/arduino-ide/commit/877c1a15591d0568fa0908c07edf375ce11c3f97).

--- a/arduino-ide-extension/src/node/grpc-client-provider.ts
+++ b/arduino-ide-extension/src/node/grpc-client-provider.ts
@@ -1,5 +1,4 @@
 import { inject, injectable, postConstruct } from 'inversify';
-import { app } from 'electron';
 import { ILogger } from '@theia/core/lib/common/logger';
 import { MaybePromise } from '@theia/core/lib/common/types';
 import { ConfigServiceImpl } from './config-service-impl';
@@ -71,10 +70,11 @@ export abstract class GrpcClientProvider<C> {
   protected abstract close(client: C): void;
 
   protected get channelOptions(): Record<string, unknown> {
+    const pjson = require('../../package.json') || { "version": "0.0.0" }
     return {
       'grpc.max_send_message_length': 512 * 1024 * 1024,
       'grpc.max_receive_message_length': 512 * 1024 * 1024,
-      'grpc.primary_user_agent': `arduino-ide/${app.getVersion()}`
+      'grpc.primary_user_agent': `arduino-ide/${pjson.version}`
     };
   }
 }

